Doc/library/enum.rst

```diff
@@ -585,9 +592,15 @@ Data Types
       Using :class:`auto` with :class:`IntFlag` results in integers that are powers
       of two, starting with ``1``.
 
-   .. versionchanged:: 3.11 :meth:`__str__` is now :func:`int.__str__` to
-      better support the *replacement of existing constants* use-case.
-      :meth:`__format__` was already :func:`int.__format__` for that same reason.
+   .. versionchanged:: 3.11
+
+      :meth:`__str__` is now :func:`int.__str__` to better support the
+      *replacement of existing constants* use-case.  :meth:`__format__` was
+      already :func:`int.__format__` for that same reason.
+
+      Inversion of a *IntFlag* now returns a positive value that is the
```

Comment:
```suggestion
      Inversion of a :class:`!IntFlag` now returns a positive value that is the
```

`IntFlag` is a class rather than a parameter (per the [docs style guide](https://devguide.python.org/documentation/markup/#quick-reference)).

Reply:
Thanks for the link, I'll give it a read.  Looks like some serious redoing of the enum docs in another PR.